demos/react/ContextDemo.tsx

```diff
@@ -48,43 +40,42 @@ export default function ContextDemo( props: ContextDemoProps ): JSX.Element {
 			<CKEditorContext
 				context={ ClassicEditor.Context as any }
 				contextWatchdog={ ClassicEditor.ContextWatchdog as any }
+				onChangeEditorsMap={ editors => {
```

Comment:
Naming things is a matter of personal taste, but for me, the name `onChangeEditorsMap` doesn't properly describe what this prop does. How about changing it to something more descriptive like `onEditorsStateChange` or similar?

Reply:
I'm not super fan of `onChangeEditorsMap` too, but I'm not sure if `onEditorsStateChange` is super clear too imho. The problem is that `onEditorsStateChange` would tell user that it tracks transitions between loading and loaded state, which is not the case in this callback. It tracks only initialized / destroyed editors.

Reply:
Other suggestions based on our conversation:

* `onEditorsUpdate`, but it might suggest that it gets called whenever editor content changes
* `onEditorsLifecycleUpdate`
* `onChangeEditorsList`

Reply:
`onChangeEditors`? What is the meaning of `Map` suffix?

Reply:
@arkflpc All editors are stored in context tree using dictionary: `name -> { instance: ... }` format.

```tsx
{
  editor1: { instance: ... },
  editor2: { instance: ... }
}
```

Reply:
`onWatchInitializedEditors` maybe more descriptive, wdyt @arkflpc ?

Reply:
Tbh, I'm also not a fan of this `map` suffix. Maybe that's just me, but I would avoid generic nouns or data types in such names. Your alternative is good. But I also think the `onChange` convention is pretty well known. So, I thought about other options, something like `onChangeInitializedEditors`.

Reply:
Renamed, imho good enough. It'll be documented anyway.